docs/dev-setup.rst

```diff
@@ -67,7 +67,7 @@ local development system:
 - `pip >= 6.1.0 <http://www.pip-installer.org/>`_
 - `virtualenv >= 1.11 <http://www.virtualenv.org/>`_
 - `virtualenvwrapper >= 3.0 <http://pypi.python.org/pypi/virtualenvwrapper>`_
-- Postgres >= 9.1 (9.3 recommended)
+- Postgres >= 9.1 (9.4 recommended)
```

Comment:
We're testing with Postgres 9.5.  Maybe we should recommend the version we test with (or vice-versa)?

Reply:
I only installed 9.5 on travis because (despite multiple attempts) I could not get a working postgis version with 9.4 there. I really don't want to spend more time just trying to get Travis to work unless we start doing other work on this project. We are running 9.4 on the RDS. I guess I could change these to 9.5 since I'm pretty sure that would work.

Reply:
That's fine, I agree it's not worth spending more time on.